src/dotnet/MLSearch.Service/Indexing/ChatContent/ChatContentIndexWorker.cs

```diff
@@ -19,6 +20,10 @@ public async Task ExecuteAsync(MLSearch_TriggerChatIndexing job, CancellationTok
         var eventCount = 0;
         var chatId = job.Id;
 
+        bool shouldBeIndexed = await chatIndexFilter.ChatShouldBeIndexed(chatId, cancellationToken).ConfigureAwait(false);
+        if (!shouldBeIndexed)
```

Comment:
We should either keep this check on job execution, or instead have it before sending 'MLSearch_TriggerChatIndexing' command to the queue.

Reply:
This will be removed as we discussed. The current solution with filters is not future proof.